<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>381</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>배경 은행계좌 등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>67</y>
     <width>56</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>은행명 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Bank_name_widget">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>65</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>66</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>계좌번호:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_no_widget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>65</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>사용부서 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depart_widget">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>98</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableWidget" name="bank_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>501</width>
     <height>192</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>은행명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>계좌번호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>사용부서</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>구분</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>사용여부</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="add_bank_list_Button">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추가저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_Button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_selected_Button">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>340</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="modify_selected_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>93</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 수정</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>구   분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>97</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Bank_name_widget</tabstop>
  <tabstop>account_no_widget</tabstop>
  <tabstop>depart_widget</tabstop>
  <tabstop>gubun_comboBox_widget</tabstop>
  <tabstop>add_bank_list_Button</tabstop>
  <tabstop>close_Button</tabstop>
  <tabstop>modify_selected_Button</tabstop>
  <tabstop>delete_selected_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>delete_selected_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_bank_list_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>bankdata_append()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>addbank_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>521</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>modify_selected_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>modify_selected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bank_view()</slot>
  <slot>addbank_close()</slot>
  <slot>bankdata_append()</slot>
  <slot>delete_selected_row()</slot>
  <slot>modify_selected()</slot>
 </slots>
</ui>
